src/beerocks/master/db/network_map.cpp

```diff
@@ -755,11 +755,11 @@ std::ptrdiff_t network_map::fill_bml_node_statistics(db &database, std::shared_p
     case beerocks::TYPE_IRE_BACKHAUL: {
         if (database.is_node_wireless(n->mac)) {
             n_type = beerocks::TYPE_CLIENT;
-            // fall to case beerocks::TYPE_CLIENT
         } else {
             break;
         }
     }
+        [[fallthrough]];
```

Comment:
Has this been clang-formatted? Indentation looks weird.

Reply:
It has - indentation looks weird due to the parenthesis which is aligned to the "case", but the code is indented - and seems that `[[fallthrough]]` is considered as code.